app/(auth)/sign-in/page.tsx

```
@@ -0,0 +1,75 @@
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+    mode: "onBlur",
+  });
+
+  const onSubmit = async (data: SignInFormData) => {
+    try {
+    } catch (error) {
+      console.log(error);
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)